hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterRpc.java

```diff
@@ -2116,6 +2121,8 @@ public void testRealUserPropagationInCallerContext()
     // Real user is added to the caller context.
     assertTrue("The audit log should contain the real user.",
         logOutput.contains(String.format("realUser:%s", realUser.getUserName())));
+    assertTrue("The audit log should contain the proxyuser port.",
+        logOutput.contains(PROXY_USER_PORT));
```

Comment:
can you assert the router port as well? There are 2 routers and it should be either of them?

Reply:
Fixed. Thanks

Reply:
@LiuGuH did you miss this? checking for the actual port in the assert, if it is the correct router port or not

Reply:
I saw it.  It needs some time for me to achieve it.  The correct router port should get from correct Namenode Server.getRemotePort via the right rpc call.

Reply:
hmm, I think then we can live without it :)